run-document-server.sh

```diff
@@ -166,6 +167,16 @@ deprecated_var() {
   fi
 }
 
+update_local_ssl() {
+  if [[ -n ${NODE_OPTIONS} ]]; then
+        for file in ${LOCAL_SSL_DIR}/ds-converter.conf ${LOCAL_SSL_DIR}/ds-docservice.conf
+          do
+          sed -i "s/environment.*/&\,NODE_OPTIONS=${NODE_OPTIONS}/g" $file
+        done
+      fi
+}
+
```

Comment:
```suggestion
update_local_ssl() {
  if [[ -n ${NODE_OPTIONS} ]]; then
    for file in ${LOCAL_SSL_DIR}/ds-converter.conf ${LOCAL_SSL_DIR}/ds-docservice.conf; do
      sed -i "s/environment.*/&\,NODE_OPTIONS=${NODE_OPTIONS}/g" $file
    done
  fi
}
```
some nicer formatting won't hurt the code